src/mds/MDCache.cc

```diff
@@ -5621,7 +5621,8 @@ void MDCache::clean_open_file_lists()
 	  dout(10) << " unlisting unwanted/capless inode " << *in << dendl;
 	  in->item_open_file.remove_myself();
 	}
-      } else if (in->last != CEPH_NOSNAP) {
+      } 
+      else {
```

Comment:
can you put } and else { on teh same line please?

Reply:
@liewegas updated